packages/example-app-base/index.html

```diff
@@ -4,43 +4,54 @@
     <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0" user-scalable="no" />
-    <title>Office UI Fabric Example App Base</title>
-    <link href="https://c.s-microsoft.com/favicon.ico?v2" rel="shortcut icon" type="image/x-icon" />
-    <style>
-      body,
-      html {
-        margin: 0;
-        padding: 0;
+    <title>UI Fabric React Examples</title>
```

Comment:
This is the code that was previously in all the other index.htmls, plus an improved way of getting the min query param.